src/www/app/outline_server_repository/server.ts

```diff
@@ -80,7 +79,7 @@ export class OutlineServer implements Server {
 
   async connect() {
     try {
-      await this.tunnel.start(this.config);
+      await this.tunnel.start(this.config.shadowsocksConnection);
```

Comment:
Rename `config` to `sessionConfig` and make it `ShadowsocksSessionConfig`. Later this will be a server config and you'll need to fetch the session config if it's a dynamic config.